internal/compiler/object_tree.rs

```diff
@@ -203,6 +203,7 @@ pub struct PopupWindow {
     pub component: Rc<Component>,
     pub x: NamedReference,
     pub y: NamedReference,
+    pub close_on_click: NamedReference,
```

Comment:
Since it has to be constant, you could also make it compile-time known maybe and use `bool` instead.

Reply:
Good idea, that'll simplify the code a fair bit.